src/etcetra/client.py

```diff
@@ -1033,6 +1034,107 @@ async def keys_range(
         )
         return [x.key.decode(encoding) for x in response.kvs]
 
+    async def election_campaign(
+        self,
+        name: bytes,
+        lease_id: int,
+        value: Optional[bytes] = None,
+    ) -> LeaderKey:
+        """
+        Campaign waits to acquire leadership in an election,
+        returning a LeaderKey representing the leadership if successful.
+        The LeaderKey can then be used to issue new values on the election,
+        transactionally guard API requests on leadership still being held,
+        and resign from the election.
+
+        Parameters
+        ---------
+        name
+            Name is the election’s identifier for the campaign.
+        lease_id
+            LeaseID is the ID of the lease attached to leadership of the election.
+            If the lease expires or is revoked before resigning leadership,
+            then the leadership is transferred to the next campaigner, if any.
+        value:
+            Value is the initial proclaimed value set when the campaigner wins the election.
+
+        Returns
+        -------
+        leader: etcetra.types.LeaderKey
+            Leader describes the resources used for holding leadereship of the election.
+        """
+        stub = v3election_pb2_grpc.ElectionStub(self.channel)
+        request = v3election_pb2.CampaignRequest(name=name, lease=lease_id, value=value)
+        response = await stub.Campaign(request)
+        return response.leader
```

Comment:
Is there a special reason election related methods doesn't support automatic value encoding/decoding feature with respect to `EtcdClient.encoding` value, like other methods? If not, then please add this feature, so that user do not have to do the coding jobs manually. It'll be great to add `encoding` keyword argument on function declaration too, since there's a chance that user will want to override default `encoding` setting when calling this method.    
Ref: https://github.com/lablup/etcetra/blob/main/src/etcetra/client.py#L282-L286